source/encode/package.json

```diff
@@ -9,9 +9,10 @@
   "scripts": {
     "test": "mocha lib/*.spec.js"
   },
-  "dependencies": {},
+  "dependencies": {
+    "aws-sdk": "^2.414.0"
```

Comment:
This is new `Rotate` is a new `VideoSelector` key, and this will fail without using the latest version of `MediaConvert`